mitosheet/mitosheet/mito_dash/v1/mito_callback.py

```diff
@@ -72,8 +72,111 @@ def new_function(*_args, **_kwargs):
             return callback(*args, **kwargs)(new_function)
         
         return function_wrapper
+    
+    def activate_mito(
+            app: Dash,
+            track_selection=False
+        ) -> None:
+        """
+        This function must be called right after instatiated your Dash application, so that it 
+        can register the mito_callback decorator with Dash.
+
+        TODO: make the Spreadsheet throw an error if the user has not called this function. I wonder
+        if we can just edit a global variable.
+
+        TODO: we could make this patch the callback function (but it might already be imported)...
+        """
+
+        @app.callback(
+            Output({'type': ID_TYPE, 'id': MATCH}, 'all_json', allow_duplicate=True), 
+            Output({'type': ID_TYPE, 'id': MATCH}, 'spreadsheet_result', allow_duplicate=True), 
+            Input({'type': ID_TYPE, 'id': MATCH}, 'message'),
+            State({'type': ID_TYPE, 'id': MATCH}, 'mito_id'),
+            State({'type': ID_TYPE, 'id': MATCH}, 'session_key'),
+            prevent_initial_call=True
+        )
+        def handle_message(msg, mito_id, session_key):
+            spreadsheet = get_component_with_mito_id(mito_id, session_key)
+            if spreadsheet is None:
+                print("No spreadsheet found for id", mito_id)
+                # TODO: should we print some error here
+                raise PreventUpdate
+            
+            spreadsheet.num_messages += 1
+
+            spreadsheet.unprocessed_messages.put(msg)
+            spreadsheet.process_single_message(session_key)
+            
+            spreadsheet.spreadsheet_result = WRONG_CALLBACK_ERROR_MESSAGE.format(prop_name='spreadsheet_result', num_messages=spreadsheet.num_messages, id=spreadsheet.mito_id, session_key=session_key)
+            return spreadsheet.get_all_json(), spreadsheet.spreadsheet_result
+        
+
+        # Because this has a performance impact, we only register this callback if
+        # the user actually uses the track_selection parameter
+        # TODO: improve the selection error message in this case...
+        if track_selection:
```

Comment:
I don't think we need this parameter. We can just not use it in the frontend....